tests/unit/store/flask.spec.js

```diff
@@ -97,14 +97,14 @@ describe("store/flask", () => {
       const bound_ping_system_status = ping_system_status.bind(context);
       await bound_ping_system_status();
 
-      expect(store.state.flask.status_uuid).toEqual(
+      expect(store.state.flask.status_uuid).toStrictEqual(
         STATUS.MESSAGE.CALIBRATION_NEEDED
       );
-      expect(store.state.playback.playback_state).toEqual(
+      expect(store.state.playback.playback_state).toStrictEqual(
         playback_module.ENUMS.PLAYBACK_STATES.CALIBRATION_NEEDED
       );
     });
-    test("Given the current state is not SERVER_READY and the returned status is having HTTP return status error 404 it should be gracefully handled and the status_uuid is set to ERROR and status_ping_interval_id is set to null ", async () => {
+    test("Given the current state is not SERVER_READY, When the status returned error 404, Then error 404 should be gracefully handled and the status_uuid is set to ERROR and status_ping_interval_id is set to null", async () => {
```

Comment:
mocking an axios route should be included in the Given section of a test title.  The "When" should be just the function the test is calling---it is pinging the system status. Whether a function returns an error shouldn't be part of the "When" section---either the "Given" section describes setting of the environment so that the function should error, or the arguments to the function when it is called would trigger an error (for example, "When calling `get_square_root` with a negative number, Then the function raises an error")